README.md
<div id="top"></div>

<!-- PROJECT LOGO -->
<br />
<div align="center">

<h3 align="center">Daily Scheduler</h3>

  <p align="center">
    Create a third-party API: Daily Scheduler
    <br />
    <a href="https://github.com/Web3-Academy-DAO/w3a-nfts"><strong>Explore the docs »</strong></a>
    <br />
    <br />
    <a href="https://github.com/Web3-Academy-DAO/w3a-nfts">View Demo</a>
    ·
    <a href="https://github.com/Web3-Academy-DAO/w3a-nfts/issues">Report Bug</a>
    ·
    <a href="https://github.com/Web3-Academy-DAO/w3a-nfts/issues">Request Feature</a>
  </p>
</div>

<!-- ABOUT THE PROJECT -->
## About The Project

[![Product Name Screen Shot][product-screenshot]](https://example.com)



<p align="right">(<a href="#top">back to top</a>)</p>



### Built With

* [![JavaScript][JavaScript.js]][JavaScript-url]
* [![JQuery][JQuery.com]][JQuery-url]
* [![Moment][Moment.js]][Moment-url]
* [![Bootstrap][Bootstrap.com]][Bootstrap-url]

<p align="right">(<a href="#top">back to top</a>)</p>

<!-- LICENSE -->
## License

Distributed under the MIT License. See `LICENSE.txt` for more information.

<p align="right">(<a href="#top">back to top</a>)</p>

[product-screenshot]: /public/assets/imgs/screenshot.png
[JavaScript.js]: https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E
[JavaScript-url]: https://www.javascript.com/
[Bootstrap.com]: https://img.shields.io/badge/bootstrap-%23563D7C.svg?style=for-the-badge&logo=bootstrap&logoColor=white
[Bootstrap-url]: https://getbootstrap.com
[JQuery.com]: https://img.shields.io/badge/jquery-%230769AD.svg?style=for-the-badge&logo=jquery&logoColor=white
[JQuery-url]: https://jquery.com
[Moment.js]: https://img.shields.io/badge/Moment-%292929AD.svg?style=for-the-badge&logo=Moment&logoColor=white
[Moment-url]: https://momentjs.com/ 

CHANGELOG.md
# Changelog
<!-- Sample
## v0.0.1

### Added or Changed
- Added this changelog :)
- Fixed typos in both templates
- Back to top links
- Added more "Built With" frameworks/libraries
- Changed table of contents to start collapsed
- Added checkboxes for major features on roadmap

### Removed
- 


### How the v0.0.0 Works
- v1.0.0  the first number represents how many times we merge into main branch.
- v0.1.0  the second number represents how many times we push or merge into dev branch.
- v0.0.1  the third number is each of our feature branch we will work out of to create each component before merging it into the dev branch.

This will keep you up to date to track progress and know what you change by adding or removing.
-->

## v0.1.0

### Added or Changed
- Uploaded starter files (Next.js, Tailwind Css) to Main and Dev Branch.  Will be working out of dev branch.

tailwind.config.js
/** @type {import('tailwindcss').Config} */
const plugin = require("tailwindcss/plugin");

const Myclass = plugin(function ({ addUtilities }) {
  addUtilities({
    ".my-rotate-y-180": {
      transform: "rotateY(180deg)",
    },
    ".preserve-3d": {
      transformStyle: "preserve-3d",
    },
    ".perspective": {
      perspective: "1000px",
    },
    ".backface-hidden": {
      backfaceVisibility: "hidden",
    },
  });
});

const conicGradient = (theme, direction, colorList) => {
  const params = [
    direction,
    ...colorList.map((color) => theme(`colors.${color}`)),
  ];

  return `conic-gradient(${params.join(", ")})`;
};

const colors = require("tailwindcss/colors");

module.exports = {
  mode: "jit",
  content: [
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {
      colors: {
        "purple": {
          DEFAULT: "#7A00FF",
          50: "#DAB8FF",
          100: "#CFA3FF",
          200: "#BA7AFF",
          300: "#A552FF",
          400: "#8F29FF",
          500: "#7A00FF",
          600: "#5F00C7",
          700: "#44008F",
          800: "#290057",
          900: "#0F001F",
        },
        amber: colors.amber,
        lightBlue: colors.lightBlue,
        rose: colors.rose,
        gray: colors.gray,
      },
      backgroundImage: (theme) => ({
        "conic-gradient": conicGradient(theme, "from 300deg", [
          "red.400",
          "amber.100",
          "lightBlue.500",
          "blue.300",
          "purple.600",
          "pink.500",
          "rose.600",
          "red.400",
        ]),
      }),
    },
  },
  plugins: [Myclass],
};


components/Backup/TestModal.js
import React, {useState} from 'react';
import Image from 'next/image';
import Logos from "/public/assets/imgs/Tier-1.png";


const TestModal =() => {
    const [tier1Modal, settier1Modal] = useState(false);

  return (
    <div className='flex md:flex-col justify-center items-center mt-40'>
        <div className='flex gap-5'>
            <button 
              type='button' 
              className='bg-blue-600 text-white active:bg-white active:text-black hover:bg-white hover:text-black flex justify-center items-center gap-2 font-bolt px-6 h-12 rounded-md shadow hover:shadown-lg outline-none focus:outline-none z-0'
              onClick={()=> settier1Modal(true)}
            >
              Open Modal
            </button>
        </div>
        {/* Tier 1 Modal UI Start */}
        {/* says show modal if not present */}
        {tier1Modal ? (
          <div className='flex justify-center items-center flex-col w-3/4 h-auto rounded-xl shadow-xl p-2 bg-black bg-opacity-75 z-50'>
              <Image src={Logos} width={300} height={400} objectFit="contain" />
              <h1 className='mt-2 text-3xl text-white'><span className='text-indigo-600'>Tier 1</span> DOer Supporter</h1>
              <h2 className='text-base mt-2 mx-4 text-white font-semibold text-center'>
                May your life be filled with success and achievements.
              </h2>
              <button 
                className='my-5 w-auto px-8 h-10 bg-blue-600 text-white reounded-md shadow hover-shadow-lg font-semibold'
                onClick={() => settier1Modal(false)}
              >
                Close
              </button>
            </div>
        ) : null}
    </div>
  )
}

export default TestModal

pages/index.js
import { useState } from "react";
import Head from "next/head";
import Image from "next/image";
import Link from "next/link";
import { SModal, CModal, LModal } from "../components/Modal";

export default function Home() {
  const [openSModal, setOpenSModal] = useState(false);
  const [openCModal, setOpenCModal] = useState(false);
  const [openLModal, setOpenLModal] = useState(false);

  return (
    <>
      <Head>
        <title>Web3Acadmey DAO Donations</title>
        <meta name="description" content="Donation page for Web3 Acadmey DAO" />
        <link rel="icon" href="/favicon.ico" />
      </Head>

      {/* Header Section */}
      <div id="stars" className="z-0"></div>
      <div id="stars2" className="z-0"></div>
      <div id="stars3" className="z-0"></div>
      <div className="px-10 py-10 md:px-20 bg-gradient-to-b from-purple-800 to-black ">
        <Image
          src="/../public/assets/imgs/h-logo.png"
          alt="Web3 Academy DAO logo"
          width="150"
          height="90"
        />
        <div className="justify-between lg:flex sm:flex-column gap-16">
          <div className="justify-center items-center">
            <h1 className="mt-20 text-center md:text-left leading-tight">
              <span className="font-light">Become a W3A DAO </span>
              <br />
              Genesis Donator
            </h1>
            <p className="mt-10 max-w-2xl text-center mb-20 md:text-left">
              Web3 Academy DAO is a tokenized community on a mission to create
              resources, products and services to better support the wave of
              entrepreneurs, businesses and creators moving into Web3 over the
              coming decade.
            </p>
          </div>
          <div className="md:w-1/2 flex-column justify-center items-center">
            <Image
              src="/../public/assets/imgs/logos.png"
              alt="DOER Icons"
              width="1000"
              height="500"
            />
            <p className="text-3xl font-light text-gray-40 text-center mt-20">
              #AreYouReady?
            </p>
          </div>
        </div>
      </div>

      {/* NFT Tier Section */}
      <h2 className="md:text-5xl sm:text-4xl max-w-xl text-center mx-auto mt-20 ">
        <span className="font-light">Web3 Academy DAO </span>
        <br />
        Genesis Donators Collection
      </h2>
      <div className="mt-10 flex-column justify-center items-center mb-20 px-5">
        <div className="w-full h-1/3 flex justify-center items-center mb-20">
          <div className="w-96">
            <h3 className="sm:text-xl md:text-lg text-purple-100 uppercase">
              Tier 1
            </h3>
            <h2 className="text-3xl">DOER Supporter</h2>
            <p className="py-4">
              A DOER Supporter holder shows that you are here for the Mission!
              Those who mint this NFT will forever be remembered as a supporter
              of the genesis of Web3 Academy DAO.
            </p>
            <button
              disabled={openSModal || openCModal || openLModal}
              className="bg-purple-500 text-white rounded-md px-8 py-2 mt-2 text-base font-bold hover:bg-purple-200 active:bg-purple-300 uppercase"
              onClick={() => setOpenSModal(true)}
            >
              Mint for 0.1 ETH
            </button>
            {openSModal && <SModal closeModal={setOpenSModal} />}
          </div>
          <div className="w-56">
            <Image
              src="/../public/assets/imgs/Tier-1.png"
              alt="/"
              width="400"
              height="450"
            />
          </div>
        </div>

        <div className="w-full h-1/3 flex justify-center items-center mb-20">
          <div className="w-56">
            <Image
              src="/../public/assets/imgs/Tier-2.png"
              alt="/"
              width="400"
              height="450"
            />
          </div>
          <div className="w-96">
            <h3 className="sm:text-xl md:text-lg text-purple-100 uppercase">
              Tier 2
            </h3>
            <h2 className="text-3xl">DOER Champion</h2>
            <p className="py-4">
              A DOER Champion holder shows that you are here for the mission and
              committed to the cause! Those who mint this NFT will forever be
              remembered as a champion of the genesis of Web3 Academy DAO.
            </p>
            <button
              disabled={openSModal || openCModal || openLModal}
              className="bg-purple-500 text-white rounded-md px-8 py-2 mt-2 text-base font-bold hover:bg-purple-200 active:bg-purple-300 uppercase"
              onClick={() => setOpenCModal(true)}
            >
              Mint for 0.25 ETH
            </button>
            {openCModal && <CModal closeModal={setOpenCModal} />}
          </div>
        </div>

        <div className="w-full h-1/3 flex justify-center items-center">
          <div className="w-96">
            <h3 className="sm:text-xl md:text-lg text-purple-100 uppercase">
              Tier 3
            </h3>
            <h2 className="text-3xl">DOER Legend</h2>
            <p className="py-4">
              A DOER Legend holder shows that you are here for the mission and
              putting your money where your mouth is! Those who mint this NFT
              will forever be remembered as a legend of the genesis of Web3
              Academy DAO.
            </p>
            <button
              disabled={openSModal || openCModal || openLModal}
              className="bg-purple-500 text-white rounded-md px-8 py-2 mt-2 text-base font-bold hover:bg-purple-200 active:bg-purple-300 uppercase"
              onClick={() => setOpenLModal(true)}
            >
              Mint for 0.5 ETH
            </button>
            {openLModal && <LModal closeModal={setOpenLModal} />}
          </div>
          <div className="w-56">
            <Image
              src="/../public/assets/imgs/Tier-3.png"
              alt="/"
              width="400"
              height="450"
            />
          </div>
        </div>
      </div>
      {/* Footer Section */}
      <div className="bg-gradient-to-t from-purple-800 to-black pb-24 pt-36 ">
        <div className="">
          <div className="text-center">
            <h1 className="text-4xl max-w-xl mx-auto">Web3 Academy DAO thanks you</h1>
            <p className="text-sm pt-2 pb-8 mt-8 max-w-xl mx-auto">
              All funds raised from this NFT mint will go directly to the DAO
              treasury, a multisig safe secured by 8 members of the DAO. The
              funds will go towards paying for various resources and tools
              needed to enable the DAO to take flight and progress towards
              supporting entrepreneurs and creators across the Web3 Ecosystem.
            </p>
          </div>
        </div>
        <div className="text-center pt-24">
          <Image
            src="/../public/assets/imgs/h-logo.png"
            alt="Web3 Academy DAO logo"
            width="150"
            height="90"
          />
        </div>
      </div>

      {/*  */}
    </>
  );
}
